### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenario_ Keyboard: "Feedback" control is not accessible using keyboard

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1115]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #NonBenchmark; #Visual Studio; #Keyboard; #WCAG2.1.1; #Reactivated;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Repro Steps:

1. Open Visual Studio Enterprise 2019 Preview 
2. Create a new .Net Core Console App (C#) 
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Verify whether "Feedback" is accessible using keyboard

### Actual Result:
"Feedback" control is not accessible using keyboard

### Expected Result:
"Feedback" control should be accessible using keyboard

### User Impact:
Keyboard only users will not be able to access feedback control

### Attachment for Reference:

[1115_ML .NET Model Builder_ feedback control is not accessible using keyboard.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5602828/1115_ML.NET.Model.Builder_.feedback.control.is.not.accessible.using.keyboard.pptx)

**Comment by beccamc** [comment]
Build to test here: https://devdiv.visualstudio.com/_apis/resources/Containers/7181026/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix.

Download and install the vsix on your Visual studio instance. This build can be used to test all the "Ready to review" bugs (should be all)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the above given build.
Now keyboard focus is going to "feedback" control. But still user is not able to perform action on the feed back control using keyboard

**Comment by beccamc** [comment]
@SaisreeDandamudi I see. So the tab problem is fixed, but this is still unusable by users. I will move back and add ability to open the menu. Thanks!

**Comment by beccamc** [comment]
Build https://devdiv.visualstudio.com/_apis/resources/Containers/7343454/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix

**Comment by SaisreeDandamudi** [comment]
Verified the bug in latest build and the issue is fixed. Hence closing the bug
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.10.0 Preview 1.0

https://user-images.githubusercontent.com/51435580/111149724-3fbe2c00-85b3-11eb-8888-6de6c4cfa0dd.mp4

**Comment by poojapatil36** [comment]
Github Tag: #Closed;
